src/rocksdb_options.rs

```diff
@@ -23,37 +23,91 @@ use merge_operator::{self, MergeOperatorCallback, MergeOperands, full_merge_call
               partial_merge_callback};
 use comparator::{self, ComparatorCallback, compare_callback};
 
-#[derive(Copy, Clone)]
```

Comment:
Pretty much any case where this line appears (esp ffi.rs) it's evidence of a reckless hack that was used to get something to compile, so as we move forward we should not hesitate to rip similar lines out wherever possible.